src/app/participant-list/participant-list.component.ts

```diff
@@ -1744,23 +1744,45 @@ export class ParticipantListComponent implements OnInit {
     if (column && column.participantColumn) {
       name = column.participantColumn.name;
     }
-    if (personData && personData.data && name) {
-      let currentKey = Object.keys(personData.data).find(key => key === name);
-      let field = personData.data[currentKey];
-      if (field) {
-        if (column.options && column.options[0] && column.options[0].name) {
-          let fieldToShow = null;
-          if (column.additionalType === Filter.ACTIVITY_STAFF_TYPE) {
-            fieldToShow = column.options.find(nameValue => nameValue.name === field);
-          } else {
-            fieldToShow = column.options.find(nameValue => nameValue.value === field);
-          }
-          return fieldToShow.name;
+    let result: string;
+    result = this.getPersonFieldFromDataRow(personData, column, name);    
+    return result;
```

Comment:
maybe just directly `return this.getPersonFieldFromDataRow(personData, column, name);` instead of declaring/assigning/returning.